src/helpers/did.rs

```diff
@@ -0,0 +1,522 @@
+use std::str::FromStr;
+
+use crate::api::{VadeEvan, VadeEvanError};
+use crate::helpers::datatypes::{
+    AddPublicKeys,
+    AddServices,
+    CreateDidPayload,
+    DIDOperationType,
+    DidUpdatePayload,
+    Patch,
+    PublicKeyJWK,
+    PublicKeyModel,
+    PublicKeyPurpose,
+    RemovePublicKeys,
+    RemoveServices,
+    Service,
+    EVAN_METHOD,
+    TYPE_BBS_KEY,
+    TYPE_JSONWEB_KEY,
+    TYPE_SIDETREE_OPTIONS,
+};
+use base64::{decode_config, encode_config, URL_SAFE_NO_PAD};
+use uuid::Uuid;
+
+pub struct DID<'a> {
+    vade_evan: &'a mut VadeEvan,
+}
+
+impl<'a> DID<'a> {
```

Comment:
Can you rename it to `Did` to avoid confusion with constants? ^^

Reply:
done